lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -346,6 +348,8 @@ def cmd_search_help
             print_line "  -o <file>         Send output to a file in csv format"
             print_line "  -S <string>       Regex pattern used to filter search results"
             print_line "  -u                Use module if there is one result"
+            print_line "  -s <column>       Sort the research results based on <column>"
+            print_line "  -r                Reverse the search_results order"
```

Comment:
```suggestion
            print_line "  -r                Reverse the order of the search results"
```

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/14917/commits/4382aeed28bccdfec455782e95fda9674f8490a4